pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -44,6 +44,11 @@ server=/crc.testing/192.168.130.11
 dns=dnsmasq
 `
 	libvirtDriverDownloadURL = fmt.Sprintf("https://github.com/code-ready/machine-driver-libvirt/releases/download/%s/crc-driver-libvirt", libvirtDriverVersion)
+	qemuBridgeConfig         = "allow crc"
+	qemuBridgeConfigPaths    = [2]string{
```

Comment:
No need for the `[2]`, as this makes code less maintainable over time: https://play.golang.org/p/PZvxl_F3bon

Reply:
I thought i tried with `[]` first but maybe i made another mistake. I'll remove the `2`.

Reply:
hence the included example to indicate this is possible